src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

```diff
@@ -14,30 +14,27 @@ redirects:
   - /docs/synthetics/synthetic-public-minion-ips
 ---
 
-New Relic uses a group of minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe, and they are in charge of actually running your monitors.
+New Relic uses public minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe and are in charge of running your monitors.
 
-Because of this, ensure your firewall allows their network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
+Because of this, ensure your firewall allows synthetics network requests. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
 
-**Recommendation:** For easier maintenance, use these options:
+Keep in mind that:
```

Comment:
Try to be more prescriptive here. A good rule of thumb is to avoid "You can" or "You should" when recommending options. Maybe rephrase this optional steps or something similar.

Reply:
Went with optional steps. When I redo this doc, I'm going to revisit this tbh.